app/bundles/UserBundle/Model/UserModel.php

```diff
@@ -331,6 +331,26 @@ private function prepareEMail(string $subject, string $content): MailHelper
         return $mailer;
     }
 
+    /**
+     * Send email with credential to the new user.
+     *
+     * @param string $email
+     * @param string $username
+     * @param string $password
+     *
+     * @return void
+     */
+    public function sendCredentialsEmail($email, $username, $password)
+    {
+        /** @var MailHelper $mailer */
+        $mailer = $this->mailHelper->getMailer();
+        $mailer->setTo($email);
+        $mailer->setSubject($this->translator->trans('mautic.email.user.new_user.subject'));
+        $text = $this->translator->trans('mautic.email.user.new_user.body', ['%username%' => $username, '%password%' => $password, '%link%' => $_SERVER['SERVER_NAME'].'/s/login']);
+        $mailer->setBody($text);
+        $mailer->send();
```

Comment:
This will conflict with https://github.com/mautic/mautic/pull/11613. I'd suggest to build this PR on top of https://github.com/mautic/mautic/pull/11613 so Symfony Mailer is used instead of Swift Mailer

Reply:
Please would you give me the replacement?

Reply:
I can point you to the docs: https://symfony.com/doc/current/mailer.html#creating-sending-messages

But it won't work until the above PR is merged. It needs some tests to be added if you'd have time to improve it.

Reply:
@alfredoct96 I can help you with the changes on this PR, please cherry pick your commits on top of #11613, I will help you with the required changes

Reply:
@mabumusa1 I don't have permissions to push on your repo